vllm/v1/structured_output/__init__.py

```diff
@@ -77,69 +84,50 @@ def _async_create_grammar(
         assert self.backend is not None
         return self.backend.compile_grammar(request_type, grammar_spec)
 
-    def grammar_bitmask(
+    def accept_tokens(self, request: Request, req_id: str,
+                      tokens: list[int]) -> bool:
+        """
+        Called in v1.core.sched.Scheduler.update_from_output after tokens have been accepted
+        Accepts a list of tokens and advances the FSM.
+        Returns True if the FSM was advanced successfully.
+        Returns False if the FSM failed to advance.
+        """
+        assert request.structured_output_request is not None and request.structured_output_request.grammar is not None
+        return request.structured_output_request.grammar.accept_tokens(
+            req_id, tokens)
+
+    def filter_logits(
         self,
-        requests: dict[str, Request],
+        input_batch: InputBatch,
+        device: torch.device,
+        scheduler_output: SchedulerOutput,
+        logits: torch.Tensor,
+        sample_hidden_states: torch.Tensor,
+    ) -> None:
+        """
+        Called in v1.worker.GPUModelRunner.execute_model immediately after the model forward pass"""
```

Comment:
nit: I think a what/why docstring should be preferred to just stating where it's used.